tests/functional/services/swift.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Client, cloudConfig, SwiftV1 } from '../../../src/oms'
+import { randomString } from '../../utils/helpers'
+
+jest.setTimeout(10000)  // for debug
+
+const authUrl = 'https://iam.eu-de.otc.t-systems.com/v3'
+const t = process.env.OS_TOKEN
+if (!t) {
+    throw 'Missing OS_TOKEN required for tests'
+}
+const config = cloudConfig(authUrl).withToken(t).config
+const client = new Client(config)
+
+beforeAll(async () => {
+    await client.authenticate()
+})
+
+test('AccountMetadata: show metadata', async () => {
+    const srv = client.getService(SwiftV1)
+    const account = await srv.showAccountMetadata()
+    expect(account).toBeDefined()
+    expect(account.domainID).toBe(client.domainID)
+})
+
+test('Account: update metadata', async () => {
+    const srv = client.getService(SwiftV1)
+    const testMeta = {
+        meta: 'data',
+        meta2: 'atad',
+    }
+    await srv.updateAccountMetadata(testMeta)
+    const account = await srv.showAccountMetadata()
+    expect(account.metadata).toHaveProperty('meta')
+    expect(account.metadata).toHaveProperty('meta2')
+    expect(account.metadata.meta).toBe(testMeta.meta)
+    expect(account.metadata.meta2).toBe(testMeta.meta2)
+})
+
+// needs a user with enough rights
+test.skip('AccountMetadata: update quota', async () => {
+    const srv = client.getService(SwiftV1)
+    const testQuota = 20
+    await srv.updateAccountMetadata(undefined, testQuota)
+    const account = await srv.showAccountMetadata()
+    expect(account.metadata['quota-bytes']).toBe(testQuota)
+})
+
+test('Containers: list', async () => {
+    const srv = client.getService(SwiftV1)
+    const containers = await srv.listContainers()
+    expect(containers).toBeDefined()
+    expect(containers).toHaveProperty('length')
+})
+
+test('Containers: workflow', async () => {
+    const srv = client.getService(SwiftV1)
+    const name = randomString(10)
+    await srv.createContainer(name, undefined, { 'container-meta': 'yes' })
+    let containers = await srv.listContainers()
```

Comment:
what about `getContainer()` against `listContainers()`

Reply:
Didn't want to put different endpoints into one test